docs/users/providers/pantheon.md

```diff
@@ -0,0 +1,47 @@
+<h1>Pantheon Hosting Provider Integration</h1>
+
+ddev provides an integration with the [Pantheon Website Management Platform](https://pantheon.io/), which allows for Pantheon users to quickly and easily download and provision a site from Pantheon in a local ddev-managed environment.
+
+## Quick Start
+
+If you have ddev installed, and have an active Pantheon account with an active WordPress, Drupal 7, or Drupal 8 site, you can follow this quick start guide to spin up a pantheon site locally.
+
+1. Authenticate with Pantheon
+    a. Login to your Pantheon Dashboard, and [Generate a Machine Token](https://pantheon.io/docs/machine-tokens/) for ddev to use.
+    b. Run `ddev auth-pantheon <YOUR TOKEN>`
```

Comment:
When you view the previewed version, the "a." and "b." are not showing up an indented child list. See preview here https://github.com/tannerjfco/ddev/blob/de1c39192703f1be2dfa41414bea64c04c99bf86/docs/users/providers/pantheon.md.